crates/forge/tests/it/zk/cheats.rs

```diff
@@ -158,14 +163,15 @@ forgetest_async!(test_zk_use_factory_dep, |prj, cmd| {
     setup_deploy_prj(&mut prj);
 
     cmd.forge_fuse();
+    // We added the optimizer flag which is now false by default so we need to set it to true
     run_zk_script_test(
         prj.root(),
         &mut cmd,
         "./script/DeployCounterWithBytecodeHash.s.sol",
         "DeployCounterWithBytecodeHash",
         Some("transmissions11/solmate@v7 OpenZeppelin/openzeppelin-contracts cyfrin/zksync-contracts"),
         2,
-        Some(&["-vvvvv", "--via-ir", "--system-mode", "true", "--broadcast"]),
+        Some(&["-vvvvv", "--via-ir", "--system-mode", "true", "--broadcast", "--optimize", "true"]),
```

Comment:
Do we need to add it everywhere in our tests?

Reply:
So why do we need it in the first place then? We can't run the test without it?